allensdk/brain_observatory/behavior/session_apis/data_transforms/behavior_data_transforms.py

```diff
@@ -243,8 +244,54 @@ def get_stimulus_templates(self) -> Optional[StimulusTemplate]:
         -------
         StimulusTemplate or None if there are no images for the experiment
         """
+
+        # TODO: Eventually the `grating_images_dict` should be provided by the
+        #       BehaviorLimsExtractor/BehaviorJsonExtractor classes.
+        #       - NJM 2021/2/23
+        grating_images_dict = {
```

Comment:
Is this lookup table really a feature of `BehaviorDataTransform`? Should it live in `behavior/stimulus_processing/__init__.py` as the default value of `grating_images_dict` in the call signature to `get_stimulus_templates`?

Reply:
That lookup table should be provided by the BehaviorLimsExtractor and BehaviorJsonExtractor classes. Since those currently do not exist, the next best place was in that method.